content/en/container-orchestration.md

```diff
@@ -0,0 +1,22 @@
+---
+title: Container Orchestration
+status: Draft
```

Comment:
I like the idea of calling it draft instead of feedback appreciated, but if we introduce that here, we have to consequently change it in all other places, especially in the localization readmes. What about doing that in separate issue/PR?!

Reply:
ok, let's talk about this in our next maintainer meeting. I think this is almost ready, so we can ignore here and update to completed once done.

Reply:
agreed